openedx_filters_samples/samples/pipeline.py

```diff
@@ -751,3 +751,29 @@ def run_filter(self, context, template_name):  # pylint: disable=arguments-diffe
         return {
             "context": context, template_name: template_name,
         }
+
+class StopAccountSettingsRender(PipelineStep):
+    """
+    Stop account settings render process raising RedirectToPage exception.
+
+    Example usage:
+
+    Add the following configurations to your configuration file:
+
+        "OPEN_EDX_FILTERS_CONFIG": {
+            "org.openedx.learning.student.settings.render.started.v1": {
+                "fail_silently": false,
+                "pipeline": [
+                    "openedx_filters_samples.samples.pipeline.StopAccountSettingsRender"
+                ]
+            }
+        },
+    """
+    def run_filter(self, context, *args, **kwargs):  # pylint: disable=arguments-differ
+        """
+        Pipeline step that stop access to account settings page.
+        """
+        raise CourseAboutRenderStarted.RedirectToPage(
```

Comment:
why redirect to page if the filter is called StopAccountSettingsRender?

Reply:
That's my bad, i just fixed it